<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGridTolerance</class>
 <widget class="QDialog" name="AddGridTolerance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>821</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius: 0px; border-bottom-right-radius: 0px; border-right-width: 0px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Value&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_reset">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>445</y>
     <width>180</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius: 0px; border-bottom-left-radius: 0px; border-left-width: 0px;</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_confirm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>445</y>
     <width>180</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius: 0px; border-bottom-right-radius: 0px; border-right-width: 0px;</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius: 0px; border-bottom-right-radius: 0px; border-right-width: 0px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textinput_time">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>641</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius: 0px; border-bottom-left-radius: 0px; border-left-width: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 5%;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Add or remove any number of point-value pairs. Both the Time and Value fields contain comma-seperated value arrays. Preview and Confirm.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_plot">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>440</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius: 0px; border-bottom-left-radius: 0px; border-left-width: 0px;</string>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="plot_points" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>451</width>
     <height>256</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="input_use_timeconfig">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>440</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius: 0px; border-bottom-right-radius: 0px; border-right-width: 0px;</string>
   </property>
   <property name="text">
    <string>Use Timeconfig</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textinput_value">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>641</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius: 0px; border-bottom-left-radius: 0px; border-left-width: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_10</zorder>
  <zorder>button_confirm</zorder>
  <zorder>button_reset</zorder>
  <zorder>label_12</zorder>
  <zorder>textinput_time</zorder>
  <zorder>label_info</zorder>
  <zorder>button_plot</zorder>
  <zorder>plot_points</zorder>
  <zorder>input_use_timeconfig</zorder>
  <zorder>textinput_value</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textinput_time</tabstop>
  <tabstop>textinput_value</tabstop>
  <tabstop>button_confirm</tabstop>
  <tabstop>button_reset</tabstop>
  <tabstop>button_plot</tabstop>
  <tabstop>input_use_timeconfig</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
